lib/containers/utils.pm

```diff
@@ -127,19 +134,23 @@ sub basic_container_tests {
     die('error: ephemeral container was not removed') if ($output_containers =~ m/test_ephemeral/);
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
-    # Containers' state can be saved to a docker image
-    my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 600);
-    die('zypper inside container timed out') if (!defined($ret));
-    if ($ret != 0) {
-        record_info('poo#40958 - curl install failure, try with force-resolution.');
-        my $output = script_output("$runtime container exec $container_name zypper in --force-resolution -y -n curl", 600);
-        die('error: curl not installed in the container') unless (($output =~ m/Installing: curl.*done/) || ($output =~ m/\'curl\' .* already installed/));
-    }
-    assert_script_run("$runtime container commit $container_name tw:saved", 240);
+    if ($runtime == 'docker' && is_sle('=15-SP1')) {
+        record_info('bsc#1190670', 'bsc#1190670 - docker blocks clone3 syscall');
+        assert_script_run("$runtime container commit $container_name tw:saved", 240);
+    } else {
+        # Containers' state can be saved to a docker image
+        my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 600);
+        die('zypper inside container timed out') if (!defined($ret));
+        if ($ret != 0) {
+            my $output = script_output("$runtime container exec $container_name zypper in --force-resolution -y -n curl", 600);
```

Comment:
Why not using --force-resolution in line 142?

Reply:
As it's not usually needed. But I'm not changing that.